src/Search/Api/Query/RamlRequirement.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+namespace eLife\Search\Api\Query;
+
+trait RamlRequirement
+{
+    public static $root = __DIR__.'/../../../../vendor/elife/api/dist/samples/';
```

Comment:
Hmm, we should use Puli rather than hardcode such a path. But I don't think anything should rely on those samples anyway.